webapp/src/main/webapp/skin/outlook.css

```diff
@@ -823,10 +828,14 @@ textarea[name='activityTitle'], textarea[name='comment'] {
 }
 
 .connections {
-    position: relative;
-    transition: 2s;
+  position: relative;
```

Comment:
Avoid reformatting where fixing one or two lines only. This mislead when need merge or backport to stable codebase.

Reply:
Fixed